Pipfile

```diff
@@ -68,3 +68,7 @@ types-pyyaml = ">=6.0"
 types-requests = ">=2.31"
 types-toml = ">=0.10"
 wheel = "==0.41.0"
+pytest-bdd = "==6.1.1"
+playwright = "==1.40.0"
```

Comment:
afaik, `playwright` is not needed when you already specify `hac-playwright`. Please confirm.

Reply:
youre right, works without playwright